poule.yml

```diff
@@ -0,0 +1,70 @@
+# For every newly created or modified issue, assign label based on matching regexp using the `label`
+# operation, as well as an Engine-specific version label using `version-label`.
+- triggers:
+      issues:       [ edited, opened, reopened ]
+  operations:
+      - type:       label
+        settings: {
+            patterns: {
+                area/builder:        [ "dockerfile", "docker build" ],
+                area/distribution:   [ "docker login", "docker logout", "docker pull", "docker push", "docker search" ],
+                area/plugins:        [ "docker plugin" ],
+                area/networking:     [ "docker network", "ipvs", "vxlan" ],
+                area/runtime:        [ "oci runtime error" ],
+                area/security/trust: [ "docker_content_trust" ],
+                area/swarm:          [ "docker node", "docker service", "docker swarm" ],
+                platform/desktop:    [ "docker for mac", "docker for windows" ],
+                platform/freebsd:    [ "freebsd" ],
+                platform/windows:    [ "nanoserver", "windowsservercore", "windows server" ],
+            }
```

Comment:
Also we might need to rethink how matching is done here: I'm using these as regular expression against the lowercase title and body of issues. Something like `"arm"` is probably too likely to match because of this :thinking: